### PR TITLE
feat(express): with logto

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Implement the middleware: withLogto.

And extract "getLogtoUserFromRequest" from Next.js SDK to Node.js SDK to share with express.

## Testing

UTs

### CODE REVIEW

File: packages/node/src/index.ts [hunk 1]
```diff
@@ -41,4 +44,40 @@ export default class LogtoClient extends BaseClient {
...
+
+  getContext = async (getAccessToken = false) => {
+    const { isAuthenticated } = this;
+
+    if (!isAuthenticated) {
+      const user: LogtoContext = {
+        isAuthenticated,
+      };
+
+      return user;
```
Comment (charIeszhao) [inline packages/node/src/index.ts:56]: nit: Return directly without declaring the `user` variable?

**Review by charIeszhao — APPROVED** [review]
Looks good

**Comment by charIeszhao** [comment]
Build failed. Need to update next sdk as well

File: packages/node/src/index.ts (hunk 1 above)
Comment (wangsijie) [reply packages/node/src/index.ts]: updated

**Pull request merged**